client/src/index.tsx

```diff
@@ -10,6 +10,87 @@ import { GleanProvider } from "./telemetry/glean-context";
 
 // import * as serviceWorker from './serviceWorker';
 
+let attemptedFallback = false;
+window.addEventListener("error", async (error) => {
```

Comment:
Don't we get an [Error]Event, rather than an error? (I was confused by seeing `error.error` below.)

```suggestion
window.addEventListener("error", async (event) => {
```